WhaleSpotting/react-app/src/styles/Card.scss

```diff
@@ -0,0 +1,62 @@
+@import "./Constants.scss";
+
+.card-component {
+    width: 100%;
+    border-radius: 10px;
+    background-color: $primaryCard;
+    height: fit-content;
+    margin: 15px 0; 
+    position: relative;
+
+    .card-info {
+        display: inline-flex;
+        padding: 10px 10px;
+        width: 100%;
+    }
+
+    .first-column {
+        width: 50%;
+    }
+
+    .second-column {
+        width: 50%;
+    }
+
+    .pending {
+        position: absolute;
+        right: 7px;
+        top: 5px;
+        width: fit-content;
+        color: $primaryDark;
+    }
+}
+
+.confirmed {
```

Comment:
as described in Card.tsx we can remove this

Reply:
removed